spec/features/home_page_spec.rb

```diff
@@ -92,4 +92,51 @@
       expect(find(@alert)).to have_content(@sign_in_msg)
     end
   end
+
+  describe 'contact us' do
+    before(:each) do
+      @name_textbox      = 'input#name'
```

Comment:
Can you use `let` instead of instance variables.  See http://www.betterspecs.org/#let

Reply:
was doing this to match what all of the other specs do and don't really want to go back and change all the specs to use let right now, do we prefer consistency or having it the preferred style in one spot?